.travis.yml

```diff
@@ -13,14 +13,15 @@ branches:
   - master
   - nightly
   - ci
+  - test/cli
```

Comment:
The `ci` branch is made for this purpose and this line is not needed.

Reply:
Oh, I forgot to remove this line. Sry.